pkg/config/config.go

```diff
@@ -1874,6 +1875,16 @@ func (c *CustomAttributeMap) Decode(value string) error {
 	return nil
 }
 
+func (c *CustomAttributeMap) ToDataMap() (d data.Map) {
```

Comment:
I'd say the `To` in the name here is redundant, `DataMap()` seems clear enough to me.
Either case I'd add a brief comment describing what it does, since it's an exported method

```suggestion
// ToDataMap returns the CustomAttributeMap as a data.Map.
func (c *CustomAttributeMap) ToDataMap() (d data.Map) {
```